redis/src/main/scala/zio/redis/api/Connection.scala

```diff
@@ -22,7 +22,7 @@ import zio.redis.Output._
 import zio.redis._
 
 trait Connection extends RedisEnvironment {
-  import Connection._
+  import Connection.{Auth => _, _}
```

Comment:
Now I'm using `Auth` (class, because of `AuthInput`). And `Connection.Auth` is basically key - `"Auth"` and I had to distinguish between those two. A similar approach is used for Keys I think.